### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: .scripts/build_steps.sh [hunk 1]
```diff
@@ -30,10 +30,11 @@ source run_conda_forge_build_setup
 make_build_number "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 
 conda build "${RECIPE_ROOT}" -m "${CI_SUPPORT}/${CONFIG}.yaml" \
+    --suppress-variables ${EXTRA_CB_OPTIONS:-} \
     --clobber-file "${CI_SUPPORT}/clobber_${CONFIG}.yaml"
 
 if [[ "${UPLOAD_PACKAGES}" != "False" ]]; then
-    upload_package "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
+    upload_package  "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
```
Comment (mkcor) [inline .scripts/build_steps.sh:37]: I guess this change wasn't intended...?

File: .azure-pipelines/azure-pipelines-linux.yml [hunk 2]
```diff
@@ -6,15 +6,21 @@ jobs:
 - job: linux
   pool:
     vmImage: ubuntu-16.04
```
Comment (mkcor) [inline .azure-pipelines/azure-pipelines-linux.yml:8]: Seems a tad old...?

**Review by mkcor — COMMENTED** [review]
I wonder why voila v0.2.1 couldn't be resolved... :thinking: 
```bash
conda.exceptions.ResolvePackageNotFound: 
  - voila[version='>=0.2.1,<0.3']
```

File: .scripts/build_steps.sh (hunk 1 above)
Comment (SylvainCorlay) [reply .scripts/build_steps.sh:37]: these changes are auto-generated by the conda-smithy.

All human-generated changes are in meta.yaml

Comment (mkcor) [reply .scripts/build_steps.sh:37]: Oh, ok. Thanks for taking care of this!

**Pull request merged**